tests/generators_test.cpp

```diff
@@ -138,7 +138,7 @@ namespace generators
         std::vector<std::unique_ptr<objects::TriangleObj>> objectsVec;
         objects::TriangleObj object({-0.6, -0.6, 1}, {0.6, -0.6, 1}, {-0.6, 0.6, 1});
 
-        core::Ray tempRay = source.GenerateRay(0, 0);
+        core::Ray tempRay = source.generateRay(0, 0);
```

Comment:
The name in the class is upper-cased `GenerateRay`.

Does this code even compile?

Reply:
generateRay builds ray object based on arguments and simulation parameters

Reply:
This is not what I asked about.

I meant that name is incorrect, as the class has it with upper-cased letter. I think this does not compile.